.travis.yml

```diff
@@ -12,6 +12,18 @@ matrix:
         - python: "3.4"
         - python: "pypy3"
         - python: "pypy"
+before_install:
+    - identify -version
+    - convert -list format
+    - sudo apt-get remove imagemagick
+    - wget http://www.imagemagick.org/download/releases/ImageMagick-6.9.1-10.tar.xz
+    - tar xf ImageMagick-6.9.1-10.tar.xz
+    - cd ImageMagick*
+    - ./configure --disable-shared --with-openjp2=yes --with-webp=yes --prefix=/usr
+    - make
+    - sudo make install
```

Comment:
Sure thing. But I am having some troubles getting everything compiled correctly.

The issue is that ImageMagick is not able to find the OpenJpeg libs for some reason, so jp2 is not listed in the delegates. And we could use the opj_compress binary that comes with installing OpenJpeg but there seems to be a bug in the last release where PNG's are not having their color space interpreted correctly. I confirmed this was the case in the issue log with OpenJpeg. It seems to have been fixed but they have not created a new release since some time in 2014 : (

So, there are a lot of ways I could get this to work but I'm going to work on getting a build that's stable. I think the safest thing to do might be to use the same version of ImageMagick that is on Ubuntu 12.04 by default. And recompile with the older libs. OpenJpeg effectively replaces libjpeg62 but libjpeg62 may be more stable.

So I'm still working on all of that. 

FYI, building a perfect version of ImageMagick with webp, jpeg2000 support and more on OSX is super easy. The Homebrew maintainers for that have done a good job.